src/Propel/Generator/Reverse/MysqlSchemaParser.php

```diff
@@ -283,6 +298,74 @@ public function getColumnFromRow($row, Table $table)
         return $column;
     }
 
+    /**
+     * Sets table and column descriptions according to source.
+     *
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return \Propel\Generator\Model\Table
+     */
+    protected function addDescriptions(Table $table): Table
+    {
+        $tableDescription = $this->loadTableDescription($table);
+        if ($tableDescription) {
+            $table->setDescription($tableDescription);
+        }
+
+        foreach ($table->getColumns() as $column) {
+            $columnDescription = $this->loadColumnDescription($column);
+            if ($columnDescription) {
+                $column->setDescription($columnDescription);
+            }
+        }
+
+        return $table;
+    }
+
+    /**
+     * Load a comment for this table.
+     *
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return string|null
+     */
+    protected function loadTableDescription(Table $table): ?string
+    {
+        $tableName = $this->getPlatform()->quote($table->getName());
+        $query = <<< EOT
+SELECT table_comment
+FROM INFORMATION_SCHEMA.TABLES
+WHERE table_schema=DATABASE()
+  AND table_name=($tableName)
+EOT;
+
+        /** @var string|null */
```

Comment:
The var looks wrong, there is a different Syntax for such cases afaik.

Reply:
I looked [here](https://psalm.dev/docs/annotating_code/supported_annotations/), but could not find a dedicated annotation for this scenario.
The only other solutions I found was to silence the psalm error in the baseline file, which I wanted to avoid. Or add a clunky type cast, which ultimately casts a string to a string. Or store the return value in an otherwise useless variable, just for the var statement.
I was surprised that var works here, and agree that it looks a bit off. But next to the other options, I found it the least bad. Let me know if you prefer another option, or know another way to do it.

Reply:
You are right, without the $variable here this seems to be the syntax for returning.
All good, sry :)